### PR TITLE
Prow testing

### PR DESCRIPTION
**Pull request by pohly**
This enables unit testing. It would be possible to write a custom E2E test for leader election, but that can be added later.

/assign @msau42

### CODE REVIEW

**Comment by pohly** [comment]
/hold

This tentatively uses https://github.com/kubernetes-csi/csi-release-tools/pull/13, which needs to be merged first. We also need https://github.com/kubernetes/test-infra/pull/12088 before this can be tested.

File: rpc/common_test.go [hunk 1]
```diff
@@ -241,7 +241,7 @@ func TestGetPluginCapabilities(t *testing.T) {
 			defer os.RemoveAll(tmp)
 			identity := &fakeIdentityServer{
 				getPluginCapabilitiesResponse: test.output,
-				err: injectedErr,
+				err:                           injectedErr,
```
Comment (pohly) [inline rpc/common_test.go]: I'm not sure why this passed testing in https://github.com/kubernetes-csi/csi-lib-utils/pull/20.

Comment (pohly) [reply rpc/common_test.go]: The explanation is that the Docker image that is currently used by the mandatory pre-merge check l only has Go 1.10.1. We'll transition to current Go as part of replacing the Prow check.

**Comment by pohly** [comment]
@msau42 are we ready to replace pull-sig-storage-csi-lib-utils-stable (the custom Prow job) with pull-sig-storage-csi-lib-utils (the generated one) and enable also mandatory unit testing for csi-release-tools and csi-test?

https://github.com/kubernetes/test-infra/blob/24aefcb76ce14366c3e4380aebcb1ec7eb3664e7/config/jobs/kubernetes-csi/gen-jobs.sh#L310-L328

I think we can and should. I'd also like to rename to `pull-kubernetes-csi-<repo>`, for the sake of consistency.

**Comment by msau42** [comment]
Sounds fine? Which prs need to go in first?

**Comment by msau42** [comment]
Did you want to rename the jobs first?

**Pull request merged**